src/dune_rules/merlin/merlin.ml

```diff
@@ -130,7 +130,7 @@ module Processed = struct
       Error
         "The current Merlin configuration has been generated by another, incompatible, \
          version of Dune. Please rebuild the project. (Using the same version of Dune as \
-         the one running the `ocaml-merlin` server.)"
+         the one running the 'ocaml-merlin' server.)"
```

Comment:
`User_message.command`?

Reply:
Same as above, this doesn't get printed to the terminal but serialized into an S-Expression. Presumably something is printing this somewhere but hardcoding this will make the output of dune a bit more consistent.

Given this doesn't prevent a better solution in the future, I don't think it is worth overthinking this too much in this PR.